src/services/travis.service.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/prefer-default-export,no-await-in-loop */
+import fetch from 'node-fetch';
+
+function mapTravisBuildToEndorBuild(item) {
+  const formattedBuild = {};
+  formattedBuild.id = item.id;
+  formattedBuild.buildNumber = item.number;
+  formattedBuild.state = item.state;
+  formattedBuild.previousState = item.previous_state;
+  formattedBuild.duration = item.duration;
+  formattedBuild.type = item.event_type;
+  formattedBuild.user = item.created_by.login;
+  formattedBuild.started_at = item.started_at;
+  formattedBuild.finished_at = item.finished_at;
+  return formattedBuild;
+}
+
+/**
+ * Gets the build statuses for a project
+ * @param projectName the project name in the format owner/projectName
+ * @param token the token
+ * @returns {Promise<void>}
+ */
+export async function getBuildStatusesForProject(projectName, token, branchName) {
```

Comment:
Is this tested anywhere?

Reply:
no, I'm going to open issues for testing these. I was never going to get this work done for the week if I wrote tests for it all.